elki/src/main/java/ch/ethz/globis/pht/pre/PreProcessorRangeD.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2011-2015 ETH Zurich. All Rights Reserved.
+ *
+ * This software is the proprietary information of ETH Zurich.
+ * Use is subject to license terms.
+ */
+package ch.ethz.globis.pht.pre;
+
+public interface PreProcessorRangeD {
+	
+	/**
+	 * 
+	 * @param raw raw data (input)
```

Comment:
I went over the javadoc and fixed several issues with missing @param / @return, see d0d1273eb786706506e44bd3cc64fdb7fd95627c